### PR TITLE
Backport trivial pr20

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: contrib/linearize/example-linearize.cfg [hunk 1]
```diff
@@ -5,6 +5,9 @@ rpcpassword=somepassword
 host=127.0.0.1
 port=9998
 
+#signet default
+#port=38332
```
Comment (PastaPastaPasta) [inline contrib/linearize/example-linearize.cfg:9]: 20015 doesn't harm anything, but should probably drop

File: contrib/signet/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+Contents
+========
+This directory contains tools related to Signet, both for running a Signet yourself and for using one.
+
+getcoins.py
+===========
+
+A script to call a faucet to get Signet coins.
```
Comment (PastaPastaPasta) [inline contrib/signet/README.md:8]: drop 20145

File: src/util/system.h [hunk 3]
```diff
@@ -90,7 +90,7 @@ bool FileCommit(FILE *file);
 bool TruncateFile(FILE *file, unsigned int length);
 int RaiseFileDescriptorLimit(int nMinFD);
 void AllocateFileRange(FILE *file, unsigned int offset, unsigned int length);
-bool RenameOver(fs::path src, fs::path dest);
+[[nodiscard]] bool RenameOver(fs::path src, fs::path dest);
```
Comment (PastaPastaPasta) [inline src/util/system.h:93]: 20519 I think this causes a warning somewhere, need to look at

File: src/test/denialofservice_tests.cpp [hunk 4]
```diff
@@ -85,7 +85,6 @@ BOOST_AUTO_TEST_CASE(outbound_slow_chain_eviction)
     dummyNode1.SetSendVersion(PROTOCOL_VERSION);
 
     peerLogic->InitializeNode(&dummyNode1);
-    dummyNode1.nVersion = 1;
```
[the remark this answers is not in the document]
Comment (PastaPastaPasta) [reply src/test/denialofservice_tests.cpp]: done, also squashed 20519

**Review by UdjinM6 — CHANGES_REQUESTED** [review]
pls drop 19733 and 19734 (merged via #4430 )

**Comment by PastaPastaPasta** [comment]
Done

**Review by UdjinM6 — APPROVED** [review]
utACK

**Pull request merged**